shipitscript/ship_actions.py

```diff
@@ -25,6 +25,22 @@ def mark_as_shipped(ship_it_instance_config, release_name):
                              status='shipped', shippedAt=shipped_at)
 
 
+def mark_as_shipped_v2(ship_it_instance_config, release_name):
+    """Function to make a simple call to Ship-it API v2 to change a release
+    status to 'shipped'
+    """
+    taskcluster_client_id, taskcluster_access_token, api_root, \
```

Comment:
Nit: maybe we can we gather all this within a tuple that we can split across two lines to improve readability? Something like

```
(taskcluster_client_id, taskcluster_access_token, 
  api_root, timeout_in_seconds) = get_auth_primitives_v2(ship_it_instance_config)
```